ktor-client/ktor-client-plugins/ktor-client-logging/api/ktor-client-logging.api

```diff
@@ -5,6 +5,7 @@ public final class io/ktor/client/plugins/logging/LogLevel : java/lang/Enum {
 	public static final field INFO Lio/ktor/client/plugins/logging/LogLevel;
 	public static final field NONE Lio/ktor/client/plugins/logging/LogLevel;
 	public final fun getBody ()Z
+	public static fun getEntries ()Lkotlin/enums/EnumEntries;
```

Comment:
New API in a patch release

Reply:
getEntries is default enum field generated by the kotlin compiler